moninag/templates/contact/verified.html

```diff
@@ -0,0 +1,8 @@
+{% extends "../index.html" %}
+
+{% block content %}
+
+  <h3>You have confirmed email recieves from MoniNag!</h3>
+  <p>Click <a href="http://127.0.0.1:8000">here</a> to visit home page of MoniNag.</p>
```

Comment:
@mhumetskyi coverage now is 100%. And u can update only one field of contact. In my case it works.